### PR TITLE
Continuation (replacement) of #279 : Making results explainable.

### PR DESCRIPTION
**Pull request by hvwaldow**

### CODE REVIEW

**Review by mhagdorn — CHANGES_REQUESTED** [review]
I think this is a great addition as it opens another dimension that RSEs need to consider that usually gets brushed under the carpet.

I think the next paragraph in the paper is in comparison relatively weak. Maybe that would be a good place to add some of the ethical concerns coming from the various biases.

File: competencies.md [hunk 1]
```diff
@@ -332,6 +332,32 @@ Further values loosely based on that code include the obligations
...
+Humphreys (2004) regards this development as "more important than the invention of the calculus in the 1660s, an event that remained unparalleled for almost 300 years".
+The epistemological status of computer modelling and simulation is still the subject of debate,
+which ranges from the postulate of a new process of knowledge creation that has its own, unique, epistemology (Winsberg, 1999, https://doi.org/10.1017/S0269889700003422)
+to the perception that from a philosophy of science perspective, there isn't really anything new (Frigg and Reiss, 2009, https://doi.org/10.1007/s11229-008-9438-z).
+In any case it is clear, that the construction of a simulation-model,
+including the selection of the salient characteristics of the system to be modelled,
+the choice of the mathematical representation of the processes to be represented,
+the choice of numerical methods and other algorithms
+and even including the design of the user-interface
+will have a decisive impact on the adequacy for purpose of the model.
```
Comment (mhagdorn) [inline competencies.md:348]: Do we need a reference for adequacy of purpose? This seems to be philosophical term with a specific meaning.

File: competencies.md [hunk 2]
```diff
@@ -332,6 +332,32 @@ Further values loosely based on that code include the obligations
...
+The relationship between initial state, inputs and final state of a computer simulation is "epistemically opaque" ( https://doi.org/10.1093/0195158709.001.0001 , Humphrey2004 ),
+in that not every step of the process is directly observable.
+The current trend of an increasing application of computationally irreducible systems, such as those based artificial neural networks,
```
Comment (CaptainSifff) [inline competencies.md:351]:
```suggestion
The current trend of an increasing application of computationally irreducible systems, such as those based on artificial neural networks,
```

Comment (CaptainSifff) [reply competencies.md:351]: I haven't read Humphreys2004, but can we claim that it's relevant still, although 20 years old?

**Pull request merged**